scripts/xjoin-config/debezium.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "xjoin.inventory.db",
+    "config": {
+      "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+      "errors.log.include.messages": "true",
+      "database.dbname": "insights",
+      "database.user": "insights",
+      "max.queue.size": "1000",
+      "transforms.unwrap.delete.handling.mode": "rewrite",
+      "slot.name": "xjoin_inventory",
+      "transforms": "unwrap",
+      "database.server.name": "xjoin.inventory",
+      "database.port": "5432",
+      "table.include.list": "public.hosts",
+      "database.hostname": "db-host-inventory",
+      "database.password": "insights",
+      "poll.interval.ms": "100",
+      "name": "xjoin.inventory.db",
+      "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState",
+      "max.batch.size": "10",
+      "errors.log.enable": "true"
```

Comment:
As per the docs below params have default values and should work? So can you remove those

[max.queue.size](https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-property-max-queue-size)
[poll.interval.ms](https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-property-poll-interval-ms)
[max.batch.size](https://debezium.io/documentation/reference/stable/connectors/postgresql.html#postgresql-property-max-batch-size)

Reply:
Done